### ISSUE TITLE
Fix package / release set

### ISSUE THREAD

**Issue by npeltier** [adobe/adobe-dx#121]
discussed with Chris, we should have fixed releases for each module (with the new structure):
- the parent should *not* have one version for each module.
- We should not release everything every time. 

With this issue i'll release everything, and point to fixed version, hopefully i also manage to remove old packages or archive them.

**Comment by npeltier** [comment]
looks like deletion of public packages is not possible and that we'll need to live with all old packages..https://docs.github.com/en/packages/publishing-and-managing-packages/deleting-a-package#about-public-package-deletion

**Comment by npeltier** [comment]
i think @auniverseaway told me about that 422 error from GH:
```[INFO] Uploading to github-dx: https://maven.pkg.github.com/adobe/adobe-dx/com/adobe/dx/admin.reactor/0.0.12/admin.reactor-0.0.12.pom.asc
[INFO] Progress (1): 833 B
[INFO]
[INFO] Uploading to github-dx: https://maven.pkg.github.com/adobe/adobe-dx/com/adobe/dx/admin.reactor/0.0.12/admin.reactor-0.0.12-source-release.zip.asc
[INFO] Progress (1): 833 B
[INFO]
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] Reactor Summary for Adobe DX - Admin - Reactor 0.0.12:
[INFO] [INFO]
[INFO] [INFO] Adobe DX - Admin - Reactor ......................... FAILURE [  7.350 s]
[INFO] [INFO] Adobe DX - Admin Components - Core bundle .......... SKIPPED
[INFO] [INFO] Adobe DX - Admin Components - App package .......... SKIPPED
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] BUILD FAILURE
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [INFO] Total time:  8.287 s
[INFO] [INFO] Finished at: 2020-07-03T11:38:02+02:00
[INFO] [INFO] ------------------------------------------------------------------------
[INFO] [WARNING] The requested profile "pom.xml" could not be activated because it does not exist.
[INFO] [ERROR] Failed to execute goal org.apache.maven.plugins:maven-deploy-plugin:3.0.0-M1:deploy (default-deploy) on project admin.reactor: ArtifactDeployerException: Failed to deploy artifacts: Could not transfer artifact com.adobe.dx:admin.reactor:pom:0.0.12 from/to github-dx (https://maven.pkg.github.com/adobe/adobe-dx): Transfer failed for https://maven.pkg.github.com/adobe/adobe-dx/com/adobe/dx/admin.reactor/0.0.12/admin.reactor-0.0.12.pom 422 Unprocessable Entity -> [Help 1]
[INFO] [ERROR]```

**Comment by npeltier** [comment]
next 422 :-)
```[INFO] [WARNING] The requested profile "pom.xml" could not be activated because it does not exist.
[INFO] [ERROR] Failed to execute goal org.apache.maven.plugins:maven-deploy-plugin:3.0.0-M1:deploy (default-deploy) on project admin.core: ArtifactDeployerException: Failed to deploy artifacts: Could not transfer artifact com.adobe.dx:admin.core:jar:0.0.12 from/to github-dx (https://maven.pkg.github.com/adobe/adobe-dx): Transfer failed for https://maven.pkg.github.com/adobe/adobe-dx/com/adobe/dx/admin.core/0.0.12/admin.core-0.0.12.jar 422 Unprocessable Entity -> [Help 1]
[INFO] [ERROR]```

**Comment by npeltier** [comment]
so it looks like the issue is with node exectution during the `release:perform` cycle, will try to fix this

**Comment by npeltier** [comment]
fixed with #125
